build.gradle

```diff
@@ -278,71 +288,67 @@ subprojects {
   // Publishing
   // ----------
 
-  artifacts {
-    archives javadocJar, sourcesJar, shadowJar, testlibJar
-  }
+  if (!nonReleaseProjects.contains(project.name)) {
+    publishing {
+      publications {
+        mavenJava(MavenPublication) {
+          version = project.version
 
-  signing {
-    required { gradle.taskGraph.hasTask("uploadArchives") }
-    sign configurations.archives
-  }
+          from components.java
 
-  if (project.hasProperty('ossrhUsername') && project.hasProperty('ossrhPassword') &&
-      !nonReleaseProjects.contains(project.name)) {
-    uploadArchives {
-      repositories {
-        mavenDeployer {
-          beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+          artifact javadocJar
+          artifact sourcesJar
+          artifact shadowJar
+          artifact testlibJar
 
-          repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-            authentication(userName: ossrhUsername, password: ossrhPassword)
-          }
-
-          snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
-            authentication(userName: ossrhUsername, password: ossrhPassword)
-          }
-
-          pom.project {
-            name 'GAX (Google Api eXtensions)'
-            packaging 'jar'
-            artifactId project.name
-            description 'Google Api eXtensions'
-            url 'https://github.com/googleapis'
+          pom {
+            name = 'GAX (Google Api eXtensions)'
+            packaging = 'jar'
+            artifactId = project.name
+            description = 'Google Api eXtensions'
+            url = 'https://github.com/googleapis'
 
             scm {
-              url 'https://github.com/googleapis/gax-java'
-              connection 'scm:git:https://github.com/googleapis/gax-java.git'
+              url = 'https://github.com/googleapis/gax-java'
+              connection = 'scm:git:https://github.com/googleapis/gax-java.git'
             }
 
             licenses {
               license {
-                name 'BSD'
-                url 'https://github.com/googleapis/gax-java/blob/master/LICENSE'
+                name = 'BSD'
+                url = 'https://github.com/googleapis/gax-java/blob/master/LICENSE'
               }
             }
 
             developers {
               developer {
-                id 'GoogleAPIs'
-                name 'GoogleAPIs'
-                email 'googleapis@googlegroups.com'
-                url 'https://github.com/googleapis'
-                organization 'Google, Inc.'
-                organizationUrl 'https://www.google.com'
+                id = 'GoogleAPIs'
```

Comment:
Does it work as before? Without Assignment operator it was calling setter methods on the delegated object ('developer' instance). With the assignment operator I'mo not even sure what is happening now.

Reply:
The `pom` declaration under `mavenJava` needs to be this way... I don't know why. (Do you feel that I need to figure it out before you approve?)